documentation/asciidoc/computers/remote-access/raspberry-pi-connect.adoc

```diff
@@ -0,0 +1,6 @@
+[[raspberry-pi-connect]]
+== Screen share with Raspberry Pi Connect
+
+You can access the desktop of a Raspberry Pi remotely from another computer or device on the same network using Raspberry Pi Connect.
```

Comment:
Seems a bit odd to mention "the same network" here - Raspberry Pi Connect "just works" with both local and remote Raspberry Pis, no matter where they are on the internet.